server/vitals.js

```diff
@@ -121,6 +121,26 @@ async function checkHeartbeat() {
   }
 }
 
+// Check for internal issues by calling DB, if there is, then log error in Sentry
+async function checkForInternalProblems() {
+  const firmwareStateMachineLocations = await db.getActiveFirmwareStateMachineLocations()
+  for (const location of firmwareStateMachineLocations) {
+    if (!location.client.isActive || !location.isActive) {
```

Comment:
Note that this test will change when we do the rebase that will replace the `is_active` flag with `is_sending_alerts`, `is_sending_vitals`, and `is_displayed`. Which one(s) do you think we should use here?

Reply:
I think we'd have to check `is_sending_vitals` to make sure it's sending vitals in the first place, and `is_sending_alerts` for both the location and the client?